examples/misc/analyzer-results-stable.txt

```diff
@@ -2,6 +2,13 @@ Analyzing misc...
 
   error - bin/cheatsheet/nullable.dart:7:13 - A value of type 'Null' can't be assigned to a variable of type 'int'. Try changing the type of the variable, or casting the right-hand type to 'int'. - invalid_assignment
   error - lib/library_tour/core/collections.dart:5:14 - The argument type 'int' can't be assigned to the parameter type 'String'. - argument_type_not_assignable
```

Comment:
When I run this locally I get 5 issues, not 10 issues. Why is that?

```
mit-macbookpro5:examples mit$ pwd
/Users/mit/dev/dart-website/examples

mit-macbookpro5:examples mit$ ~/sdk/dart/2.15.0/bin/dart analyze misc
Analyzing misc...                      1.7s

   info • lib/effective_dart/design_good.dart:204:25 • Dead code. Try removing the code, or fixing the code before it so that it can be
          reached. • dead_code
   info • lib/effective_dart/usage_good.dart:493:9 • Use initializing formals when possible. • prefer_initializing_formals
   info • lib/language_tour/built_in_types.dart:83:12 • Dead code. Try removing the code, or fixing the code before it so that it can be
          reached. • dead_code
   info • test/language_tour/built_in_types_test.dart:116:24 • Dead code. Try removing the code, or fixing the code before it so that it can
          be reached. • dead_code
   info • test/language_tour/operators_test.dart:18:15 • Dead code. Try removing the code, or fixing the code before it so that it can be
          reached. • dead_code

5 issues found.
```

Reply:
Sorry, I don't understand what that means... What analysis command does CI run (given it produces more issue, not less, than what I see with `dart analyze`)?

Reply:
Sure, the related code that is ran in CI is here: https://github.com/dart-lang/site-www/blob/master/tool/analyze-and-test-examples.sh#L38-L131

As an example, at https://github.com/dart-lang/site-www/blob/master/examples/misc/lib/language_tour/variables.dart#L112 the `unnecessary_type_check` diagnostic is ignored like:

```dart
// ignore: stable, beta, dev, unnecessary_type_check
```

The CI script will disable this ignore for the stable, beta, and dev channels so it shows up in the related analyzer result files.